src/screens/Feed.js

```diff
@@ -1,124 +1,122 @@
+            <SettingsButton icon="horizontal" />
+          </View>
+          <Post
+            img={image}
+            isLiked={true}
+            likes={hearts}
+            isBookmarked={true}
+            caption={caption}
+            username={username}
+            commentsLength={0}
+          />
+          <Divider />
         </View>
-        <Post
-          img={img}
-          isLiked={isLiked}
-          likes={likes}
-          isBookmarked={isBookmarked}
-          caption={caption}
-          username={username}
-          commentsLength={comments.length}
-        />
-      </View>
+      </TouchableWithoutFeedback>
     );
   };
 
   //sample post data for ui testing purposes. We also have use react-native carousel for swiping posts.
   render() {
-    const posts = [
-      {
-        id: 0,
-        img:
-          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
-        caption:
-          'Quarantine selfie #coronasa dsajl as lnsaln aln flsan lsan lfaslf fnasl nfals nlsan lsan lasn lfasn lfasnf lan',
-        likes: 2,
-        saves: 0,
-        comments: [
-          {
-            userName: 'Model1234',
-            userImage:
-              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
-            comment: 'Slay girllll!!!',
-          },
-        ],
-        username: 'gabriela',
-        likers: {
-          dennyliang: true,
-        },
-        bookmarkers: {
-          dennyliang: false,
-        },
-      },
-      {
-        id: 1,
-        img:
-          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
-        caption: 'Quarantine selfie #corona',
-        likes: 2,
-        saves: 0,
-        comments: [
-          {
-            userName: 'Model1234',
-            userImage:
-              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
-            comment: 'Slay girllll!!!',
-          },
-        ],
-        username: 'gabriela',
-        likers: {
-          dennyliang: false,
-        },
-        bookmarkers: {},
-      },
-      {
-        id: 2,
-        img:
-          'https://www.familyandmedia.eu/wp-content/uploads/2018/10/529382-4386816-selfie-psicologia-725x545.jpg',
-        caption: 'Quarantine selfie #corona',
-        likes: 2,
-        saves: 0,
-        comments: [
-          {
-            userName: 'Model1234',
-            userImage:
-              'https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
-            comment: 'Slay girllll!!!',
-          },
-        ],
-        username: 'gabriela',
-        likers: {},
-        bookmarkers: {
-          dennyliang: true,
-        },
-      },
-    ];
+    console.log('Posts are:', this.state.posts);
```

Comment:
Why are we logging the posts? Is this just for development purposes? Remove if not needed